client/js/components/month/month-header.js

```diff
@@ -8,12 +8,12 @@ var MonthHeader = React.createClass({
   render: function () {
     return (
       <div className="date-bar">
-        <div className="arrow arrow-left"></div>
-        <h2>May 2015</h2>
-        <div className="arrow arrow-right"></div>
+        <PrevMonth className="arrow arrow-left" month={this.props.month} year={this.props.year} />
```

Comment:
classNames should only exist on the render method of the component. if you want them to be dynamic, you could pass them in as props.